storybook/stories/core/components/List/list-schema/table.list.ts

```diff
@@ -1,13 +1,15 @@
-import leftPanel from "./left-panel.table.list";
-import topBar from "./top-bar.table.list";
-import headers from "./headers.table.list";
+import leftPanel from './left-panel.table.list';
+import topBar from './top-bar.table.list';
+import headers from './headers.table.list';
 
-import { ListTable } from "@orangehrm/oxd/core/components/List/types";
+import {ListTable} from '../../../../../../components/src/core/components/List/types';
```

Comment:
I think the way the types are reffered is wrong.

Reply:
This was  an error in the storybook I wanted to fix since I saw it when I go there. It is outside of the components. It is not necessary anyways there. forgot to remove the line. I will remove that